sdk/authorization/microsoft-azure-authentication-msi-token-provider/migration-guide.md

```diff
@@ -0,0 +1,99 @@
+
+### Cross Service SDK improvements
+
+The modern Azure identity client library also provides the ability to share in some cross-service improvements made to the Azure development experience, such as
+
+- A unified authentication approach between clients.
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries.
+- A unified asynchronous programming model using [Project Reactor][project-reactor].
+- A unified method of creating clients via client builders to interact with Azure services.
+
+## Important changes
+The Azure Identity library provides [Azure Active Directory (AAD)](https://docs.microsoft.com/azure/active-directory/fundamentals/active-directory-whatis) token authentication support across the Azure SDK. It provides a set of TokenCredential implementations which support AAD token authentication for Azure SDK clients.
+This includes [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) support for Azure Virtual Machines and Azure App Services.
+
+### Group id, artifact id, and package names
+
+Group ids, artifact ids, and package names for the modern Azure client libraries for Java have changed. They follow the [Java SDK naming guidelines][GuidelinesJavaDesign]. Each will have the group id `com.azure`, an artifact id following the pattern `azure-[area]-[service]`, and the root package name `com.azure.[area].[Service]`. The legacy clients have a group id `com.microsoft.azure` and their package names followed the pattern `com.microsoft. Azure.[service]`. This provides a quick and accessible means to help understand, at a glance, whether you are using modern or legacy clients.
+
+In azure-identity, the modern client libraries have packages and namespaces that begin with `com.azure.identity`. The legacy library has package names starting with `com.microsoft.azure`.
+
+### Authentication with managed identity
+`microsoft-azure-authentication-msi-token-provider` allows authenticating using system and user assigned managed identity. It supports Azure App Services and Azure Virtual Machines.
+You can use `azure-identity` to do the same in the following way.
+
+
+#### Authentication with system assigned managed identity
+
+```java
+/**
+ * Authenticate with a User Assigned Managed identity.
+ */
+public void createManagedIdentityCredential() {
+    ManagedIdentityCredential managedIdentityCredential = new ManagedIdentityCredentialBuilder()
+        .clientId("<USER ASSIGNED MANAGED IDENTITY CLIENT ID>") // only required for user assigned
+        .build();
+
+    // Azure SDK client builders accept the credential as a parameter
+    SecretClient client = new SecretClientBuilder()
+        .vaultUrl("https://{YOUR_VAULT_NAME}.vault.azure.net")
+        .credential(managedIdentityCredential)
+        .buildClient();
+}
+```
+
+#### Authentication with user assigned managed identity
```

Comment:
This title doesn't match the sample below, either swap the title or sample with above.